telegram/ext/jobqueue.py

```diff
@@ -195,6 +210,46 @@ def run_repeating(self, callback, interval, first=None, context=None, name=None)
         self._put(job, time_spec=first)
         return job
 
+    def run_monthly(self, callback, when, day, context=None, name=None):
+        """Creates a new ``Job`` that runs on a monthly basis and adds it to the queue.
+
+        Args:
+            callback (:obj:`callable`): The callback function that should be executed by the new
+                job. It should take ``bot, job`` as parameters, where ``job`` is the
+                :class:`telegram.ext.Job` instance. It can be used to access its ``Job.context``
+                or change it to a repeating job.
+            when (:obj:`datetime.time`): Time of day at which the job should run. If the timezone
+                (``when.tzinfo``) is ``None``, UTC will be assumed.
+            day (:obj:`int`): Defines the day of the month whereby the job would run. It should
+                be within the range of 1 and 31, inclusive.
```

Comment:
What happens, for `day = 31` in February, April, June etc? I'd suggest another parameter `strict` or `day_is_strict`. If `strict` is `True` the job will only be executed in months that have a 31th day. Of `strict` is `False` and `day` is greater than the current months number of days, it will be executed on the last day of the month.

Reply:
done!

Reply:
done!